src/commands/kitty-review.rs

```diff
@@ -0,0 +1,70 @@
+use serenity::builder::CreateApplicationCommand;
+use serenity::model::prelude::interaction::application_command::ApplicationCommandInteraction;
+use serenity::model::prelude::interaction::InteractionResponseType;
+use serenity::prelude::Context;
+use serenity::utils::Color;
+use tracing::error;
+use i18n::t;
+use reqwest::header;
+
+pub(crate) fn register(
+    command: &mut CreateApplicationCommand,
+) -> &mut CreateApplicationCommand {
+    command
+        .name("kitty-review")
+        .description(t!("commands.kitty-review.description").to_string())
+        .description_localized("cs", t!("commands.kitty-review.description", "cs"))
+        .dm_permission(true)
+}
+
+fn get_message(locale: String) -> &'static str {
+    match locale.as_str() {
+        "cs" => t!("commands.kitty-review.message", "cs"),
+        _ => t!("commands.kitty-review.message"),
+    }
+}
+
+pub(crate) async fn run(ctx: &Context, command: &ApplicationCommandInteraction) {
+    let locale = command.clone().locale;
+
+    let tenor_api_key = "API KEY ZDE";
+    let search_url = "https://tenor.googleapis.com/v2/search?q=kitty-review&key=".to_owned() + tenor_api_key;
+    let response = reqwest::Client::new()
+        .get(&search_url)
+        .header(header::USER_AGENT, "SSPS-KB BOT")
```

Comment:
`User-Agent: <product> / <product-version> <comment>`
```suggestion
        .header(header::USER_AGENT, "SSPS-KB/1.0 workshop-bot")
```